lib/ohai/config.rb

```diff
@@ -16,28 +18,91 @@
 # limitations under the License.
 #
 
-require 'mixlib/config'
+require 'ohai/log'
+require 'chef-config/logger'
+
+ChefConfig::Logger = Ohai::Log
+
+require 'chef-config/config'
 
 module Ohai
+  Config = ChefConfig::Config
```

Comment:
This is a little confusing. Is this about to monkeypatch ChefConfig::Config?
A comment might be helpful here.